packages/teams-js/src/public/pages.ts

```diff
@@ -34,9 +34,9 @@ export namespace pages {
   /**
    * @hidden
    *
-   * Registers a handler when focus needs to be passed from teams to the place of choice on app.
+   * Registers a handler when focus needs to be passed from the host to the place of choice on application.
```

Comment:
Since we're in the file, could we wordsmith a bit? How about: 
Registers a handler when focus passes from the host to a specified part of the application